File: HealthCatalyst.Web.UI/src/app/search/model/PersonResult.ts
export class PersonResult {
  name: string;
  interests: string;
  age: number;
  address: string;
  pictureFileName: string;
  constructor(
    name: string,
    interest: string,
    age: number,
    address: string,
    pictureFileName: string
  ) {
    this.name = name;
    this.interests = interest;
    this.age = age;
    this.address = address;
    this.pictureFileName = pictureFileName;
  }
}

File: HealthCatalyst.Services/Models/PersonResult.cs
using System;

namespace HealthCatalyst.Services.Models
{
    public class PersonResult
    {
        public string PictureFileName { get; set; }        
        public string Name { get; set; }
        public string Address { get; set; }
        public int Age { get; set; }
        public string Interests { get; set; }
    }
}
